projects/igniteui-angular/src/lib/grids/common/events.ts

```diff
@@ -20,7 +20,7 @@ export interface IGridCellEventArgs extends IBaseEventArgs {
     event: Event;
 }
 
-export interface IGridEditEventArgs extends CancelableEventArgs, IBaseEventArgs {
+export interface IGridBaseEditEventArgs extends IBaseEventArgs {
```

Comment:
why not `IGridDoneEditEventArgs`

Reply:
I was going for it, yet initially it seemed a bit off to me, for the "ing" event with IGridEditEventArgs to extend IGridDoneEditEventArgs. On second thought is seems I am OK with both names.